data/hai/hai reveal 5 philosophers.txt

```diff
@@ -0,0 +1,1075 @@
+			`	You explain to him about the upgrades the Unfettered need to their ships in order to join battle, and the plan to rescue the Hai and end the standoff before anyone does anything stupid.`
+			`	"And of course," he says, "you're not an official representative of the Republic, so you need someone to authorise and organise things if it's to happen quickly."`
+			`	"Well have no fear, I can definitely do that!" Within a few short minutes you seem to have lifted a handful of years off of Xilin's worried features.`
+			`	"I'll sort out the approvals, and I'm sure the Marshal can sort out the logistics. Give me the details and we'll have it all waiting for you on Prime by the time you arrive."`
+			`	You thank him, and as he seems eager to get that ball rolling you leave him to return to your ship.`
+
+mission "Hai Rescue [C02] Escort to Arms Purchase"
+	landing
+	name "Missing Researchers"
+	description "Escort the Unfettered Hai Shield Beetles to <destination>. All must arrive intact. Payment is <payment>."
+	source "Allhome"
+	destination "Prime"
+	to offer
+		has "Hai Rescue [C01] Bloody Love Blooms: done"
+	on visit
+		dialog `To finish this mission, all eleven Unfettered Hai Shield Beetles must land with you.`
+	on enter Betelgeuse
+		dialog `The large fleet of human ships in this system fly the same flag, "Human Alliance," and appear to have outfits from all factions. The human authorities must be concerned about well-equipped Unfettered Hai Shield Beetles wandering through human space. Hopefully this human camaraderie will outlive the current crisis.`
+	on offer
+		conversation
+			`Teeneep comes to you wearing her fencing outfit, still stained with blood, though you notice there's more blood now, in different places than before. She departs from an eleventh Shield Beetle; you can see weapons sticking out that are not of Hai or human origin. Its thrusters are so hot they're still glowing red.`
+			`	She says, "I've had a good time with the Unfettered," probably referring to her blood stains, "but now we must advance our plans. We need to reach <destination> immediately. Neither the Fettered Hai nor the Republic want this fleet in their space for long. You'll be paid <payment> when we arrive."`
+				accept
+	npc save accompany
+		government "Hai (Friendly Unfettered)"
+		personality escort disables plunders harvests
+		ship "Shield Beetle (Sunbeam Korath Jump)" "Unfettered Flagship"
+		fleet 10
+			names "hai"
+			variant
+				"Shield Beetle (Fuel Jump)"
+	npc
+		government "Human Alliance"
+		personality escort heroic staying opportunistic
+		system Betelgeuse
+		fleet "Human Wormhole Guards" 20
+	on complete
+		payment 600000
+		conversation
+			`As you come in to land the local Republic officials inform you that the outfitters have emptied ten large bays so nobody notices the Hai on the ships. There are few planets with such a large excess capacity for capital ship outfitter bays; no wonder the Hai picked Prime.`
```

Comment:
It may make more sense for Xilin to have picked Prime, as he would have a more detailed understanding of human space than the Hai in this context.